source/extensions/filters/common/fault/BUILD

```diff
@@ -17,6 +17,7 @@ envoy_cc_library(
         "//source/common/http:headers_lib",
         "//source/common/protobuf:utility_lib",
         "@envoy_api//envoy/extensions/filters/common/fault/v3:pkg_cc_proto",
+        "@envoy_api//envoy/extensions/filters/http/fault/v3:pkg_cc_proto",
```

Comment:
this import looks wrong -> importing `/http` from `/common`

Reply:
I think this is due to transitive include but I'm not sure. @htuch ?

Reply:
Since both paths are explicitly referenced in `fault_config.cc`, we explicitly treat them as distinct deps here, I don't think there is a correctness issue.